### PR TITLE
short-term load pipeline finished with dockerfile.

### PR DESCRIPTION
**Pull request by olliejth**
Created load and pipeline files and Dockerfile for creating images. Also added to requirements.

### CODE REVIEW

File: short_term/load.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from extract import extract_recordings
+from boto3 import client
+
+
+def get_s3_client(env: dict):
+    """Returns boto3 s3 client."""
+    return client(service_name="s3",
+                  aws_access_key_id=env["AWS_ACCESS_KEY"],
+                  aws_secret_access_key=env["AWS_PRIVATE_ACCESS_KEY"])
```
Comment (mehrdadhalali) [inline short_term/load.py:10]: Whitespace

File: short_term/load.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from extract import extract_recordings
```
Comment (mehrdadhalali) [inline short_term/load.py]: Module docstring plz

File: short_term/load.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+from boto3 import client
```
Comment (mehrdadhalali) [inline short_term/load.py:3]: Put this import first.

File: short_term/load.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def get_s3_client(env: dict):
+    """Returns boto3 s3 client."""
+    return client(service_name="s3",
+                  aws_access_key_id=env["AWS_ACCESS_KEY"],
+                  aws_secret_access_key=env["AWS_PRIVATE_ACCESS_KEY"])
+
+
+def upload_csv(filename, s3_client, env) -> None:
```
Comment (mehrdadhalali) [inline short_term/load.py:13]: Docstring missing

File: short_term/load.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    bucket = env["BUCKET_NAME"]
+    upload_filename = f"recordings/{filename}"
+
+    s3_client.upload_file(filename, bucket, upload_filename)
+
+
+def load_recordings(env: dict, file_name: str) -> None:
```
Comment (mehrdadhalali) [inline short_term/load.py:21]: Docstring plz

File: short_term/pipeline.py
Comment (mehrdadhalali) [inline short_term/pipeline.py:1]: Docstrings plz

File: short_term/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Specifies latest image of python
+FROM python:latest
+
+# Copies txt file containing env package requirements
+COPY requirements.txt .
+
+# Pip installs required packages.
```
Comment (Leon-JS02) [inline short_term/Dockerfile:7]: Could we use a WORK_DIR?

**Review by mehrdadhalali — APPROVED** [review]

File: short_term/pipeline.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+def main(env: dict) -> None:
+    """Extracts and uploads API data to S3."""
+    file_name = extract_recordings()
+    load_recordings(env, file_name)
+
+
+if __name__ == "__main__":
+
+    load_dotenv()
+
```
Comment (Leon-JS02) [inline short_term/pipeline.py:19]: Do you need to pass `ENV` into the function? Could it not be called from within?

**Review by YakMan101 — APPROVED** [review]
Looks like should work. Will need some changes later

**Pull request merged**